lib/rules/no-duplicate-selectors/index.mjs

```diff
@@ -112,6 +112,12 @@ const rule = (primary, secondaryOptions) => {
 		 * @param {Map<string, { line: number }>} contextSelectorSet
 		 */
 		function checkSelectorListInDifferentRules(ruleNode, contextSelectorSet) {
+			const flattenedNestedSelectors = flattenNestedSelectorsForRule(ruleNode, result).flatMap(
```

Comment:
refactor
l122–124

```
				nodes: flattenedNestedSelectors,
```

Reply:
Good catch, thanks.

I added the early check, but forgot to use the variable here.